### PR TITLE
Added the capitalize tool

### PR DESCRIPTION
**Pull request by Akatsuki06**

### CODE REVIEW

File: tool/settings.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
```
Comment (AdiChat) [inline tool/settings.py]: Add `STATIC_ROOT = os.path.join(BASE_DIR, 'static')` to avoid any errors while `collectstatic`.

File: tool/settings.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'luyvbu@6mjvtmr9#ch#p)tves=e$$+il_5sz*c_yy=1*ujja9c'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (AdiChat) [inline tool/settings.py]: Change `DEBUG` to `false`. :+1:

File: tool/settings.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+ALLOWED_HOSTS = []
```
Comment (AdiChat) [inline tool/settings.py]: Change to `ALLOWED_HOSTS = ['localhost']`

File: templates/base.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+<head>
+    <title> Tools</title>
+
+    <meta name="description" content="">
+    <meta name="keywords" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```
Comment (AdiChat) [inline templates/base.html]: Move scripts to end of file.

**Comment by Akatsuki06** [comment]
i have done the changes and  generalized the tools

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus_first

**Pull request merged**